app/server/src/main/scala/org/bitcoins/server/BitcoindRpcAppConfig.scala

```diff
@@ -137,16 +147,25 @@ case class BitcoindRpcAppConfig(
     ZmqConfig(zmqHashBlock, zmqRawBlock, zmqHashTx, zmqRawTx)
 
   lazy val bitcoindInstance: BitcoindInstance = {
-    val fallbackBinary =
-      if (isRemote) BitcoindInstance.remoteFilePath else DEFAULT_BINARY_PATH
+    // val fallbackBinary =
+    // if (isRemote) BitcoindInstance.remoteFilePath else DEFAULT_BINARY_PATH
 
     BitcoindInstance(
       network = network,
       uri = uri,
       rpcUri = rpcUri,
       authCredentials = authCredentials,
       zmqConfig = zmqConfig,
-      binary = binaryOpt.getOrElse(fallbackBinary),
+      binary = binaryOpt match {
+        case Some(file) => file
+        case None => {
+          val homeVar = sys.env("HOME");
+          val config = ConfigFactory
```

Comment:
Is there a reason this config reference doesn't work? 

https://github.com/bitcoin-s/bitcoin-s/blob/4f7b6422ea3390f276733ee3c0fb21d426bde751/app-commons/src/main/scala/org/bitcoins/commons/config/AppConfig.scala#L120

Reply:
AppConfig already has a 'network' and a 'datadir'. Plus, if I extend AppConfig, I'd have to implement all of it's methods here. So, I wasn't too sure if I should use it

Reply:
It already extends `AppConfig`

https://github.com/shruthii625/bitcoin-s/blob/2fa4dddbcc673fd21c71ec55c09c00ab88f17da6/app/server/src/main/scala/org/bitcoins/server/BitcoindRpcAppConfig.scala#L25

Reply:
I hadn't properly read your message and assumed you were talking about BitcoindInstance yikes
I've changed it to use the config of AppConfig and it seems to be working fine